docs/installation/install-binary.md

```diff
@@ -987,4 +988,173 @@ kubectl patch CustomResourceDefinition resourcebindings.work.karmada.io --patch-
 kubectl patch CustomResourceDefinition clusterresourcebindings.work.karmada.io --patch-file webhook_in_clusterresourcebindings.yaml
 ```
 
+## Install karmada-scheduler-estimator
 
+karmada-scheduler use gRPC to access karmada-scheduler-estimator. You can use "Linux Virtual Server" as the Load Balancer.
+
+You need to deploy above components and join member cluster into Karmada first. See: [Install Karmada on your own cluster](./installation.md#install-karmada-on-your-own-cluster)
+
+### Create Systemd Service
+
+In the example below, "/etc/karmada/physical-machine-karmada-member-1.kubeconfig" is kubeconfig file of the joined member cluster.
+
+
+
+Execute operations at `karmada-01` `karmada-02` `karmada-03`.  Take `karmada-01` as an example.
+
+/usr/lib/systemd/system/karmada-scheduler-estimator.service
+
+```
+[Unit]
+Description=Karmada Scheduler Estimator
+Documentation=https://github.com/karmada-io/karmada
+
+[Service]
+# You need to change `--cluster-name` `--kubeconfig`
+ExecStart=/usr/local/sbin/karmada-scheduler-estimator \
+  --cluster-name "physical-machine-karmada-member-1" \
+  --kubeconfig "/etc/karmada/physical-machine-karmada-member-1.kubeconfig" \
+  --logtostderr=true \
+  --server-port 10352 \
+
+Restart=on-failure
+RestartSec=5
+LimitNOFILE=65536
+
+[Install]
+WantedBy=multi-user.target
+```
+
+### Start karmada-scheduler-estimator
+
+```
+systemctl daemon-reload
+systemctl enable karmada-scheduler-estimator.service
+systemctl start karmada-scheduler-estimator.service
+systemctl status karmada-scheduler-estimator.service
+```
+
+### Create Service
+
+(1) Create `karmada-scheduler-estimator.yaml `
+
+```
+apiVersion: v1
+kind: Service
+metadata:
+  name: karmada-scheduler-estimator-{{MEMBER_CLUSTER_NAME}}
+  namespace: karmada-system
+  labels:
+    cluster: {{MEMBER_CLUSTER_NAME}}
+spec:
+  ports:
+    - protocol: TCP
+      port: {{PORT}}
+      targetPort: {{TARGET_PORT}}
+  type: ExternalName
+  externalName: {{EXTERNAL_NAME}}
+```
+
+`{{PORT}}`: "--scheduler-estimator-port" parameter of "karmada-scheduler".
+
+`{{TARGET_PORT}}`: LoadBalancer IP.
+
+`{{EXTERNAL_NAME}}`: LoadBalancer Host.
+
+`{{MEMBER_CLUSTER_NAME}}`: Member Cluster Name
+
+
+
+(2) Create Service
+
+```
+kubectl create --kubeconfig "/etc/karmada/admin.kubeconfig" -f karmada-scheduler-estimator.yaml 
+```
+
+### Verify
+
+```
+$ ./check_status.sh
+###### Start check karmada-scheduler-estimator
+ok
+###### karmada-scheduler-estimator check success
+```
+
+### Trouble Shooting
+
+`karmada-scheduler` takes some time to find the new `karmada-scheduler-estimator`. If you can't wait, you can just restart karmada-scheduler.
```

Comment:
1. `karmada-scheduler` takes some time to find the xxxx ？ give a order number

Reply:
OK